src/modules/profile/profile-filter.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateUserDto {
+  @ApiProperty({ required: false })
+  search?: string;
+
+  @ApiProperty({ required: false })
+  category?: string;
+
+  @ApiProperty({ required: false })
+  skills?: string;
+
+  @ApiProperty({ required: false })
+  page?: string;
+
+  @ApiProperty({ required: false })
+  sort?: string;
```

Comment:
why we need sort when create user?

Reply:
I use sort for sorting by price freelancer and its field not required only display query. Table display only query

Reply:
I rename this table to findUserdto